CONTRIBUTING.md

```diff
@@ -8,15 +8,15 @@ Follow the golder rule:
 
 ## First Time
 
-The first thing to do always is to install the extension and note how it works. One of the best ways to do this is to launch the application and go through the files that it launches in the sequential manner.
+The first thing to do always is to install the extension and note how it works. One of the best ways to do this is to launch the application and go through the files that it launches sequentially.
 
 Also take a note of the coding style and dependencies that the project follows.
 
 A helpful resource for this project is https://developer.chrome.com/extensions/faq
 
 ## Choosing issues
 
-It is recommended to go through the issues tab once you have ran the program once and look for easy issues.
+It is recommended to go through the issue tabs, then run the program once and look for the easy issues.
```

Comment:
Hey Aditi, I think this modification changes the order in which a potential contributor is to contribute. Can you please have a look?

Reply:
hey, @thealphadollar! sorry, I just missed it. I'll fix it for sure. Thanks for your help!